readthedocs/search/models.py

```diff
@@ -137,3 +137,60 @@ def generate_distribution_of_top_queries(cls, project_slug, n):
             final_data['int_data'].append(count_of_other)
 
         return final_data
+
+
+class PageView(TimeStampedModel):
```

Comment:
I don't think this should use `TimeStampedModel` -- we only want a `date` field on it, not a created & updated DateTime field.